packages/dds/counter/api-report/counter.api.md

```diff
@@ -31,8 +31,8 @@ export interface ISharedCounterEvents extends ISharedObjectEvents {
 export class SharedCounter extends SharedObject<ISharedCounterEvents> implements ISharedCounter {
     constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes);
     protected applyStashedOp(op: unknown): void;
-    static create(runtime: IFluidDataStoreRuntime, id?: string): SharedCounter;
-    static getFactory(): IChannelFactory;
+    static create(runtime: IFluidDataStoreRuntime, id?: string): ISharedCounter;
```

Comment:
This looks like it might need a call out as a breaking change?

Reply:
Done. As this API is only alpha, its not a huge deal, but you are correct it should be noted!